deploy.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash -eu
+
+# deploy_continuous_load - deploys the load injector and target application as well as
+# the ServiceMonitor and PodMonitor objects that are used to supply Prometheus with 
```

Comment:
great docs, can we replicate / move most of it to the documentation of the helm chart, as i think that'll be the user interface